packages/wallets/src/routes/Router.tsx

```diff
@@ -30,6 +32,7 @@ const Router: React.FC = () => {
 
     return (
         <Switch>
+            <Route component={CompareAccountsRoute} path={`${prefix}/compare-accounts`} />
```

Comment:
@arshad-rao-deriv are you sure we need a route for this? Did you try to use modal for this?